modules/account-lib/src/coin/sol/transaction.ts

```diff
@@ -0,0 +1,20 @@
+import { BaseTransaction } from '../baseCoin';
+import { BaseKey } from '../baseCoin/iface';
+import { NotImplementedError } from '../baseCoin/errors';
+
+export class Transaction extends BaseTransaction {
+  /** @inheritdoc */
+  canSign(key: BaseKey): boolean {
```

Comment:
Added all of them except:
- signSingle    > sign method will take 1 or multiple KeyPairs
- signMultiSig
- estimateSize   > We have no use for this here since the serialize step in the solana SDK does this check. https://github.com/solana-labs/solana-web3.js/blob/master/src/transaction.ts#L614-L616